pom.xml

```diff
@@ -3,22 +3,24 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opentestsystem.delivery</groupId>
   <artifactId>tds-dll-master</artifactId>
-  <version>0.0.1-SNAPSHOT</version>
+  <version>3.0.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>TDS Data Logic Layer Master</name>
-  
-  <scm>
-        <connection>scm:git:https://github.com/SmarterApp/TestDeliverySystemDataAccess</connection>
-        <developerConnection>scm:git:https://github.com/SmarterApp/TestDeliverySystemDataAccess</developerConnection>
-        <url>https://github.com/SmarterApp/TestDeliverySystemDataAccess</url>
-   </scm>
+
+    <scm>
+        <connection>scm:git:https://github.com/SmarterApp/TDS_TestDeliverySystemDataAccess.git</connection>
+        <developerConnection>scm:git:https://github.com/SmarterApp/TDS_TestDeliverySystemDataAccess.git</developerConnection>
+        <url>https://github.com/SmarterApp/TDS_TestDeliverySystemDataAccess</url>
+        <tag>HEAD</tag>
+    </scm>
   
   <modules>
   	<module>tds-dll-api</module>
   	<module>tds-dll-mssql</module>
   	<module>tds-dll-mysql</module>
 	<module>tds-dll-common</module>
     <module>tds-dll-schemas</module>
+	<module>tds-dll-integration-tests</module>
```

Comment:
I added this module that was previously ignored.  It looks like it's just a test package, but the AllTests suite was annotated with an \@Ignore.  There were some compile-time issues that I fixed, but I didn't enable the tests.